settings.gradle

```
@@ -4,6 +4,8 @@ apply from: "${rootDir}/gradle/scripts/settings-flags.gradle"
 
 // Published BOM projects
 includeWithFlags ':bom',                                 'bom', 'publish'
+// Published version catalog projects
+includeWithFlags ':version-catalogs',                    'version-catalogs'
```

Q) Is there a reason you have removed the `publish` tag?

```suggestion
includeWithFlags ':version-catalogs',                    'version-catalogs', 'publish'
```

---

Opps, it is cruft. I tested how pom.xml changed without `publish`  😅